docs/DeveloperGuide.md

```diff
@@ -168,6 +168,46 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 ## **Implementation**
 
 This section describes some noteworthy details on how certain features are implemented.
+### \[Insert Numbering\] Add feature
+The Edit feature is facilitated by `LogicManager`. The `AddCommandParser` parses the command arguments, and returns
+an `AddCommand` that is executed by the `LogicManager`.
+
+This feature allows the user to add a new Customer.
+
+**Below is a sample usage and how the add sequence behaves at each step.**
+
+1. User chooses the Customer he/ she wants to add and enters the command `add n/Bob p/12345678 e/johnd@example.com r/5000 t/GOLD t/MEMBER`
```

Comment:
Just a small formatting issue, perhaps the space can be removed between he/she

Reply:
Resolved, thanks for spotting it!